ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/96066_TL_legal-requirements/105392_legal-requirements.js

```diff
@@ -0,0 +1,120 @@
+/// <reference types ='Cypress'/>
+
+// TO DO: Check Legal Requirement validation on first time use; check Empty tags.
+
+let url = Cypress.env('url') + '/task-list/2054'
```

Comment:
Not a huge issue, but this could be made a const instead as it's never changed in the class.